### DISCUSSION TITLE
composer update failed for enterprise package

### DISCUSSION THREAD

**Discussion by CardSys** [pimcore/pimcore#13119] [Q&A / Support]
We have an enterprise license. Now I wanted to update the Enterprise packages, but have no success:

`COMPOSER_MEMORY_LIMIT=-1 composer update pimcore/direct-edit`

results in:
```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires pimcore/direct-edit ^1.2, found pimcore/direct-edit[v1.2.4] in the lock file but not in remote repositories, make sure you avoid updating this package to keep the one from the lock file.
```

But **portal-engine update was working**:
`COMPOSER_MEMORY_LIMIT=-1 composer update pimcore/portal-engine` 

any ideas?

**Comment by mcop1** [comment]
@CardSys thanks for the information, we already noticed the problem and are currently looking into it. I will update this post, if I have any news on it

**Reply by brusch** [reply]
@CardSys Unfortunately we had a problem with our Composer repository, which is fixed now. 
Sorry for the circumstances.

**Reply by CardSys** [reply]
Thx for info;
`composer update` now working as expected; 
e.g. 

> Upgrading pimcore/direct-edit (v1.2.4 => v1.2.5)
